4.figures/supp_figure_1/README.md

```diff
@@ -1,11 +0,0 @@
-# Creating supplemental figure 1 - Plate facet morphology differences at single-cell and well-population levels
```

Comment:
ohh, gotcha, these files are deleted because the sup figure moved position

Reply:
Correct! Lots of moving around lol